.github/workflows/ci-tests.yml

```diff
@@ -0,0 +1,31 @@
+name: CI Tests
+
+on: push
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        sudo apt-get install postgresql-10-postgis-2.4
+        pip install -U pip pytest
+        pip install -r requirements.txt -r test_requirements.txt
+        pip install .
+      env:
```

Comment:
Super minor, but can the `env` section be above the `run` section? Might help it read better since the `run` commands would utilize the `env`.